azurelinuxagent/common/cgroupconfigurator.py

```diff
@@ -482,7 +503,6 @@ def enable(self):
                     "Attempted to enable cgroups, but they are not supported on the current platform")
             self._agent_cgroups_enabled = True
             self._extensions_cgroups_enabled = True
-            self.__set_cpu_quota(conf.get_agent_cpu_quota())
```

Comment:
Now Agent enable should be independent of cpu quota. We should enable even if any one of the controller (cpu or memory) is mounted/enabled.